fission/src/mirabuf/MirabufParser.ts

```diff
@@ -72,11 +73,13 @@ class MirabufParser {
         return this._rootNode
     }
 
-    public constructor(assembly: mirabuf.Assembly) {
+    public constructor(assembly: mirabuf.Assembly, progressHandle?: ProgressHandle) {
         this._assembly = assembly
         this._errors = new Array<ParseError>()
         this._globalTransforms = new Map()
 
+        progressHandle?.Update("Parsing assembly...", 0.3)
+
```

Comment:
Would it be better to move this to MirabufSceneObject instead?

Reply:
The point of that update is to indicate that the assembly is being parsed, so no.

Reply:
Also, the MirabufSceneObject doesn't exist at that point